src/battle-scene.ts

```diff
@@ -1549,7 +1549,7 @@ export default class BattleScene extends SceneBase {
     this.scoreText.setVisible(this.gameMode.isDaily);
   }
 
-  updateAndShowText(duration: integer): void {
+  updateAndShowText(duration: integer, isDaily?: boolean): void {
```

Comment:
while you are at it, please replace `integer` with `number` (integer is just a *useless* alias for number)

```suggestion
  updateAndShowText(duration: number, isDaily?: boolean): void {
```